hack/install.sh

```diff
@@ -144,24 +111,10 @@ install_faasd() {
 
 install_caddy() {
   if [ ! -z "${FAASD_DOMAIN}" ]; then
-    arch=$(uname -m)
-    case $arch in
-    x86_64 | amd64)
-      suffix="amd64"
-      ;;
-    aarch64)
-      suffix=arm64
-      ;;
-    armv7l)
-      suffix=armv7
-      ;;
-    *)
-      echo "Unsupported architecture $arch"
-      exit 1
-      ;;
-    esac
-
-    curl -sSL "https://github.com/caddyserver/caddy/releases/download/v2.4.3/caddy_2.4.3_linux_${suffix}.tar.gz" | $SUDO tar -xvz -C /usr/bin/ caddy
+    CADDY_VER=v2.4.3
+    arkade get caddy -v ${CADDY_VER}
+    $SUDO install -m 755 $HOME/.arkade/bin/faas-cli /usr/local/bin/
```

Comment:
You install the faas-cli instead of caddy.
The systemd file expects caddy to be in `/usr/bin` not `/usr/local/bin/`

```suggestion
    $SUDO install -m 755 $HOME/.arkade/bin/caddy /usr/bin/
```

Reply:
Sorry, possibly missed to save. As i have tested the script code into multipass installation. Have corrected it.